app/models/itinerary.rb

```diff
@@ -3,4 +3,8 @@ class Itinerary < ApplicationRecord
   has_many :activities
   has_many :itinerary_notes
   has_many_attached :photos
+  validates :start_date, presence: true
+  validates :end_date, presence: true
+  validates :name, presence: true
```

Comment:
do you think itineraries should also have unique names? or it's ok if 2 different itineraries have the same name?

Reply:
its ok, we can change it later